src/hooks/useSeriesData.ts

```diff
@@ -11,18 +11,18 @@ type Data = {
 };
 
 export const useSeriesData = (
-  entityId?: string,
+  seriesId: string | undefined,
```

Comment:
As mentioned a couple other places, it may be confusing having both seriesId and mediaId inputs for series functions. I think having just 1 seriesId param and here, maybe just a `useLegacySeriesFlow` flag?

Reply:
It looks like for `useSeriesData` we should better use the approach with two inputs. Here are the steps:

1. We open `MediaSeries` page using the old flow and the new flow. We don't really now which one is a right one. It is also possible that a customer uses a new flow but still has old series custom params associated with a media item.
2. So we path both props to SeriesRedirect and decide which `seriesId` an episode page will use based on the new flow response.